### DISCUSSION TITLE
Encryption Question - Third Party Tools

### DISCUSSION THREAD

**Discussion by jimcampbell100** [iterate-ch/cyberduck#15604] [Q&A]
Hi,

If I use Cryptomator to perform a client-side encryption of an S3 mounted bucket, would I be able to use another (non-mountainduck) solution to read the files in that bucket?

Also - if I used Cloudberry Drive (for example) to encrypt files in an S3 bucket, would I be able to use mountainduck to access those files (assuming I have the passphrase)?

Sorry - I know this is basic, but I'm not sure if there is a set protocol for S3 client side encryption.

Thanks!

**Comment by dkocher** [comment]
The implementation of the Cryptomator vault format in Cyberduck and Mountain Duck is interoperable with applications from Cryptomator [^1]. 

If the encryption in CloudBerry is using AWS S3 SSE [^2] it is interoperable as well. Refer to our [documentation](https://docs.cyberduck.io/protocols/s3/#server-side-encryption-sse).

[^1]: https://cryptomator.org/downloads/ 
[^2]: https://docs.aws.amazon.com/AmazonS3/latest/userguide/UsingServerSideEncryption.html
